### PR TITLE
Removed NeoTheology mentions + Misc Eris Fixes.

### PR DESCRIPTION
**Pull request by SirSiggles**
## About The Pull Request

Helps to make the server more compliant by removing Neo Theology mentions + helping to fix a problem in #12 

## Details

<details>
  <summary>Expand</summary>
Removed as many NeoTheology mentions references as I could and replaced them with lore compliant stuff. Also fixed an Asters I missed. 
</details>

## Screenshots

<details>
  <summary>Expand</summary>

</details>

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -521,8 +521,8 @@
 	icon_state = "captain"
 
 /obj/item/toy/figure/cargotech
-	name = "Guild Technician action figure"
-	desc = "A \"Space Life\" brand Guild Technician action figure."
+	name = "Unio Technician action figure"
```
Comment (EvilJackCarver) [inline code/game/objects/items/toys.dm:524]: Missed a letter

File: code/game/objects/structures/signs.dm [hunk 2]
```diff
@@ -536,27 +536,27 @@
...
-	name = "NeoTheology"
-	desc = "the Tau Cross - symbol of NeoTheology."
+	name = "Mekhane"
+	desc = "the Tau Cross - symbol of the Children of Mekhane."
 	icon_state = "neotheology"
 
 /obj/structure/sign/faction/neotheology_old
-	name = "NeoTheology"
+	name = "Mekhane"
 	desc = "the Tau Cross - symbol of NeoTheology."
```
Comment (EvilJackCarver) [inline code/game/objects/structures/signs.dm:554]: Although I don't expect it to be used in the map, there's an artifact mention of NeoTheo here.

Comment (SirSiggles) [reply code/game/objects/structures/signs.dm:554]: Ah shit, my bad.

**Comment by SirSiggles** [comment]
Also, on the plot hole forming, as far as I remember. Right now it's a stopgap measure until can fix stuff like cloning still being in the church. Gonna' close up to make those edits now though.

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (SirSiggles) [reply code/game/objects/items/toys.dm:524]: Thanks. Also gonna' get some of the action figure stuff I missed.

**Pull request merged**